mlflow/transformers.py

```diff
@@ -96,10 +102,11 @@ def _model_packages(model) -> List[str]:
     :return: A list of strings representing the underlying engine-specific dependencies
     """
     engine = _get_engine_type(model)
+    default_deps = ["accelerate"]
```

Comment:
Do we always need `accelerate`? Tensorflow models don't need it, right?

Reply:
Yeah I can move it inside the torch flag

Reply:
Sounds good :)